### DISCUSSION TITLE
Option for accented characters to be treated as not accented in the search

### DISCUSSION THREAD

**Discussion by perdix1975** [sumatrapdfreader/sumatrapdf#4293] [Ideas]
Sumatrapdf is a great reader!
It would be great if accented characters are ignored in the search. For example, if I search for ι, also ί, ϊ, ΐ would show up in the search results.

**Comment by kjk** [comment]
I believe we do that. Can you share a test PDF and exact search phrase that doesn't work for you?

**Comment by GitHubRulesOK** [comment]
@kjk 
SumatraPDF has always ignored upper/lowercase shift but it would not be as easy to determine what is a similar character globally.
Do you include ligatures like `fi` and how many `I` s are there in Un**i**code variability. Turkish Arabian just European ?
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/9789d85f-45f0-411c-b46c-0bd427158ed4)


 
[Option for accented characters to be treated as not accented in the search · sumatrapdfreader_sumatrapdf · Discussion #4293.pdf](https://github.com/user-attachments/files/15521312/Option.for.accented.characters.to.be.treated.as.not.accented.in.the.search.sumatrapdfreader_sumatrapdf.Discussion.4293.pdf)

**Comment by kjk** [comment]
Ok, I looked at the code. We do some tweaks for search but apparently not for accented characters.
